.github/services/s3/ceph_rados_s3/action.yml

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: ceph_rados_s3
+description: 'Behavior test for CEPH OBJECT GATEWAY S3.'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Ceph Rados Server
+      shell: bash
+      working-directory: fixtures/s3/ceph-rados
+      run: |
+        # Create Docker network
+        echo "Creating Docker network..."
+        docker network create --driver bridge ceph-cluster-net
```

Comment:
docker-compose should be able to setup this for this.

Reply:
will do